requirements/system-requirements.md

```diff
@@ -1,22 +1,22 @@
 ---
 title: System Requirements
 ---
-# System Requirements
+This article lists the minimum hardware and software requirements for the installation of the SPDocKit. Depending on the scenario if you wish to use SPDocKit for SharePoint On-Premises or SharePoint Online view the corresponding section below.
```

Comment:
This article lists the minimum hardware and software requirements for SPDocKit. View the corresponding section depending on the scenario you need - either SPDocKit for SharePoint On-Premises or SharePoint Online.

[the article for SPDocKit doesn't sound necessary, at least to me, and we don't usually use it; this seems like a more natural phrase, mentioning installation just makes it clunky, if you need/want to, I'd suggest something along the lines of ... for the SPDocKit installation.]